### PR TITLE
Create popup  for category instead of create delete and details page;

### PR DESCRIPTION
**Pull request by Otabek7667a**
I create popup for category  instead pages

### CODE REVIEW

**Comment by jamshidbek7375** [comment]
It would be great if you could also show pictures of your work.

**Comment by Otabek7667a** [comment]
Ok . Thanks for informatsion

Chor, 16-okt, 2024, 23:21 Jamshidbek ***@***.***>:

File: ExpenseTracker/ExpenseTracker/Controllers/CategoriesController.cs [hunk 1]
```diff
@@ -24,73 +24,73 @@ public IActionResult Index([FromQuery] GetCategoriesRequest request)
...
+
+
     [HttpPost]
     [ValidateAntiForgeryToken]
     public IActionResult Create([FromForm] CreateCategoryRequest request)
     {
         if (!ModelState.IsValid)
         {
-            return View(request);
+            // Return a JSON response with errors
```
Comment (davron41) [inline ExpenseTracker/ExpenseTracker/Controllers/CategoriesController.cs:46]: remove unneccessary comments

File: ExpenseTracker/ExpenseTracker/Views/Categories/Index.cshtml [hunk 2]
```diff
@@ -87,5 +89,328 @@
...
+
+<<script src="https://code.jquery.com/jquery-3.6.0.min.js"> </script>
+
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"> </script>
+
+
+
+
+
+
```
Comment (davron41) [inline ExpenseTracker/ExpenseTracker/Views/Categories/Index.cshtml:105]: remove redundant empty lines

**Pull request closed without merging**